hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/symmetric/SecretKeyManager.java

```diff
@@ -126,6 +127,8 @@ public synchronized boolean checkAndRotate() throws SCMException {
       state.updateKeys(updatedKeys);
       return true;
     }
+    LOG.info("The latest key was created at: " + currentKey.getCreationTime() +
```

Comment:
We may not need this log as INFO, as it'll blood the log files with not-very-useful messages. Note that this function is called by a periodic schedule.

Reply:
Understood, will remove this log message, thanks.